blogs.json

```
@@ -4843,6 +4843,13 @@
             "site_url": "https://medium.com/movingui",
             "feed_url": "https://medium.com/feed/movingui"
           },
+          {
+            "title": "Lemona Tech Blog",
+            "author": “Mohammad Ihmouda“,
+            "site_url": "https://blog.lemona.tech/",
+            "feed_url": "https://blog.lemona.tech/",
```

This also needs to be the URL to an RSS feed. It looks like this will work:

```suggestion
            "feed_url": "https://blog.lemona.tech/rss.xml",
```